examples/usecases/usecase.tf

```diff
@@ -199,7 +217,7 @@ resource "tanzu-mission-control_cluster" "create_tkg_vsphere_cluster" {
           }
 
           info {
-            name        = "default-nodepool"
+            name        = "md-0"
```

Comment:
The creation fails with `default-node pool` name is it? It seems to be computed for TKGm vSphere and AWS (default node pool creation)

Reply:
Node Pool Name
1. TKGm vSphere and AWS - computed
2. TKGs - required